src/contractTest/java/uk/gov/hmcts/reform/managecase/consumer/CCDConsumerTest.java

```diff
@@ -0,0 +1,201 @@
+package uk.gov.hmcts.reform.managecase.consumer;
+
+import au.com.dius.pact.consumer.MockServer;
+import au.com.dius.pact.consumer.dsl.PactDslJsonBody;
+import au.com.dius.pact.consumer.dsl.PactDslWithProvider;
+import au.com.dius.pact.consumer.junit5.PactConsumerTestExt;
+import au.com.dius.pact.consumer.junit5.PactTestFor;
+import au.com.dius.pact.core.model.RequestResponsePact;
+import au.com.dius.pact.core.model.annotations.Pact;
+import com.google.common.collect.ImmutableMap;
+import io.restassured.RestAssured;
+import io.restassured.path.json.JsonPath;
+import org.apache.http.entity.ContentType;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@ExtendWith(PactConsumerTestExt.class)
+public class CCDConsumerTest {
+
+    private static final String SERVICE_AUTHORIZATION = "ServiceAuthorization";
+
+    private static final String IDAM_OAUTH2_TOKEN = "pact-test-idam-token";
+    private static final String SERVICE_AUTHORIZATION_TOKEN = "pact-test-s2s-token";
+
+    static Map<String, String> headers = ImmutableMap.of(
+        HttpHeaders.AUTHORIZATION, IDAM_OAUTH2_TOKEN,
+        SERVICE_AUTHORIZATION, SERVICE_AUTHORIZATION_TOKEN
+    );
+
+    private static final String CASE_ASSIGNMENT_REQUEST = "{"
+        + "\"case_users\": ["
+                + " {\"organisation_id\": \"Test_Org\", \"case_id\": \"1588234985453946\", \"user_id\": \"abc123\","
+                + " \"case_role\": \"[Collaborator]\" }"
+            + "]"
+        + "}";
+
+    @Pact(provider = "ccd", consumer = "mca_consumer")
```

Comment:
Should we find a way to tag a method like this so that we can easily match it to its implementation on data store?
How is using **F**eature ids?